gcovr/writer/html.py

```diff
@@ -517,6 +565,35 @@ def source_row_branch(branches):
     }
 
 
+def source_row_decision(decisions):
+    if not decisions:
+        return None
+
+    taken = 0
+    uncheckable = False
+    total = 0
+    items = []
+
+    for decision_id in sorted(decisions):
+        decision = decisions[decision_id]
+        if decision.is_covered:
+            taken += 1
+        total += 1
+        items.append({
+            'taken': decision.is_covered,
+            'uncheckable': decision.is_uncheckable,
+            'name': decision_id,
+            'count': decision.count,
+        })
+
+    return {
+        'taken': taken,
+        'uncheckable': uncheckable,
+        'total': total,
+        'decisions': items,
+    }
```

Comment:
Should add a `name` field here to simplify the template. I think the value of that field would be `name = 'true' if decision_id % 2 == 0 else 'false'`.